.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         id: cache-pip
         with:
           path: .venv
-          key: ${{ runner.os }}-pip-${{ hashFiles('requirements*.txt') }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements*.txt') }}-x
```

Comment:
TODO remove this again later. for some reason the github actions cache with the usual key is corrupted and there's no way to clear that cache...